pkg/apis/convert/conversion_helper.go

```diff
@@ -19,17 +19,33 @@ package convert
 import (
 	"context"
 
-	pkgduckv1 "knative.dev/pkg/apis/duck/v1"
-
 	duckv1 "github.com/google/knative-gcp/pkg/apis/duck/v1"
 	duckv1alpha1 "github.com/google/knative-gcp/pkg/apis/duck/v1alpha1"
 	duckv1beta1 "github.com/google/knative-gcp/pkg/apis/duck/v1beta1"
+	corev1 "k8s.io/api/core/v1"
 	eventingduckv1alpha1 "knative.dev/eventing/pkg/apis/duck/v1alpha1"
 	eventingduckv1beta1 "knative.dev/eventing/pkg/apis/duck/v1beta1"
+	"knative.dev/pkg/apis"
+	pkgduckv1 "knative.dev/pkg/apis/duck/v1"
 	pkgduckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
 	pkgduckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
 )
 
+const (
+	deprecatedV1Alpha1Reason = "v1alpha1Deprecated"
+	deprecatedV1Alpha1Msg    = "V1alpha1 Sources will be deprecated after 0.18 cut."
```

Comment:
maybe something like: v1alpha has been deprecated and will be removed in 0.18. WDYT?

Reply:
Yes, this is more explicit. I will change it.